yetanotherco/aligned_layer
# feat (operator): get verification data batch from cloudfront

# Description

- Operator gets verification data from Cloudflare instead of from fs

# To test

1. Have anvil, aggregator & operator running
2. Run `make send_plonk_bls12_381_proof`
3. Operator should get batch from s3

You can see file content yourself by downloading batch file from [here](https://storage.alignedlayer.com/b4b654a31b43c7b5711206eea7d44f884ece1fe7164b478fa16215be77dc84cb.json)
